packages/react-search-ui-views/src/styles/themes/reference-ui/modules/_variables.scss

```diff
@@ -30,7 +30,8 @@ $fieldHoverColor: #f6f6f6 !default;
 $resultKeyTextColor: #777777 !default;
 
 // Font Variables
-$fontFamily: 'Roboto', sans-serif !default;
+$fontFamily: system, -apple-system, ".SFNSText-Regular", "San Francisco",
+  "Roboto", "Segoe UI", "Helvetica Neue", "Lucida Grande", sans-serif !default;
```

Comment:
I think I'm actually going to push back a bit on Lucida Grande and Helvetica Neue here. Those are generally used by older Mac systems that I don't think it makes a ton of sense to support/target if we're not supporting non-evergreen browsers in any case.

My suggestion would be to take a leaf from what GitHub/Medium/WordPress are doing for their system fonts snippets (I'm assuming that they have more resources than us to do a bunch of OS testing!), which is shorter and closer to:

```
-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```

Reply:
BTW - we definitely need to avoid just the `system` font-family. We want either `-apple-system` or `system-ui` (usually just one is fine), but not `system`. [Medium actually ran into this a while ago](https://medium.design/system-shock-6b1dc6d6596f) where `system` on Windows is an old and incredibly retro/pixelated font lol.

<img src="https://cdn-images-1.medium.com/max/2400/1*17_mbafyinp64GNZHjuhtg@2x.png">

Reply:
@zumwalt Do you have an opinion here?

Constance, what is the full font-family value you are proposing.

I don't hav an opinion, I will put whatever you want in here.

Reply:
```
-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```
Would be my suggestion.

Reply:
I agree with @constancecchen.